kolibri/core/content/api.py

```diff
@@ -176,6 +179,17 @@ class Meta:
             "contains_quiz",
         ]
 
+    def filter_random(self, queryset, name, value):
+        """
+        Return the content randomly ordered
+
+        :param queryset: all content nodes for this channel
+        :param value: a random value, it's not used in the query but it's used
+                      by the cache system, so same value could return same order
+        :return: content nodes randomly ordered
+        """
+        return queryset.order_by("?")
```

Comment:
I assume there's no need of a guarantee of persistent randomization between different clients for the same 'seed' value here?

Reply:
No, there is no need to fetch the same results passing the same seed value.